docs/tutorials/geomesa-quickstart-kafka-streams.rst

```diff
@@ -0,0 +1,349 @@
+GeoMesa Kafka Streams Quick Start
+=========================
```

Comment:
```suggestion
GeoMesa Kafka Streams Quick Start
============================
```

Reply:
hmm, the alignment looks right in the editor but not in the diff... anyway, rst is picky about the header indicator matching the length of the title.

Reply:
i think the suggestion is right now...